src/redis/rdb.h

```diff
@@ -111,6 +111,7 @@
 /* Range 200-240 is used by Dragonfly specific opcodes */
 
 /* Special RDB opcodes (saved/loaded with rdbSaveType/rdbLoadType). */
+#define RDB_OPCODE_DF_MASK    245  /* Mask for key properties */
```

Comment:
line 111: /* Range 200-240 is used by Dragonfly specific opcodes */

Reply:
ok thx, I misunderstood your last comment in the ticket